docs/devices/701721.md

```diff
@@ -126,7 +126,14 @@ The minimal value is `5` and the maximum value is `35`.
 The unit of this value is `°C`.
 
 ### External_measured_room_sensor (numeric)
-If `radiator_covered` is `true`: Set at maximum 30 minutes interval but not more often than every 5 minutes and 0.1 degrees difference. Resets every 35 minutes to standard. If `radiator_covered` is `false`: Set at maximum 3 hours interval but not more often than every 30 minutes and 0.1 degrees difference. Resets every 3 hours to standard. Value 21C = 2100 (-8000=undefined)..
+The temperature sensor of the TRV is — due to its design — relatively close to the heat source (i.e. the hot water in the radiator). Thus there are situations where the `local_temperature` measured by the TRV is not accurate enough: If the radiator is covered behind curtains or furniture, if the room is rather big, or if the radiator itself is big and the flow temperature is high, then the temperature in the room may easily diverge from the `local_temperature` measured by the TRV by 5°C to 8°C. In this case you might choose to use an external room sensor and send the measured value of the external room sensor to the `External_measured_room_sensor` property.
+
+The way the TRV operates on the `External_measured_room_sensor` depends on the setting of the `Radiator_covered` property:
+
+If `Radiator_covered` is `true` (Auto Offset Mode): You *must* set the `External_measured_room_sensor` property *at least* every 3 hours. After 3 hours the TRV disables this function and resets the value of the `External_measured_room_sensor` property to -8000 (disabled). You *should* set the `External_measured_room_sensor` property *at most* every 30 minutes or every 0.1K change in measured room temperature.
```

Comment:
0.1K … i doubt kelvin here is correct.

Reply:
Well Kelvin is what the Danfoss documents say. In the end a difference in Kelvin it’s the same as the difference in Celsius (same scale, different ‚offset‘). But if for the sake of clarity and understandability, it should probably be changed to 0.1C.